src/os_maild/os_maild_client.c

```diff
@@ -672,3 +612,70 @@ MailMsg *OS_RecvMailQ_JSON(file_queue *fileq, MailConfig *Mail, MailMsg **msg_sm
 
     return NULL;
 }
+
+
+
+void cJSON_PrintTable(cJSON *item, char *printed, size_t *body_size, char *tab, int dep)
+{
+    char *val1 = NULL, *val2 = NULL;
+    int log_size;
+    char *t = malloc(14*sizeof(char));
```

Comment:
Why 14?

Reply:
It's the maximum number of times it can be tabulated.